src/components/PostList/PostList.jsx

```diff
@@ -1 +1,11 @@
-export const PostList = () => <>Put the list here</>;
+import { PostInfo } from '../PostInfo/PostInfo';
+
+export const PostList = ({ posts }) => (
+  <>
```

Comment:
what is the purpose of this fragment?

Reply:
This is the component that returns JSX file. Which will create the necessary markup for each element. Next, React renders the elements one by one on our page.